src/status_im/ui2/screens/chat/composer/reply.cljs

```diff
@@ -19,6 +19,19 @@
       (focus-input-on-reply reply had-reply text-input-ref)
       (when reply
         [rn/view
-         {:style {:padding-horizontal 15
-                  :padding-vertical   8}}
-         [reply/reply-message reply true]]))))
+         {:style (merge
+                  {:padding-horizontal 15
+                   :padding-vertical   8}
+                  (when @input/recording-audio?
+                    {:position :absolute
+                     :top      12
+                     :left     0
+                     :right    0
+                     ;When recording an audio and replying at the same time,
```

Comment:
@briansztamfater, could you change this multiline comment to use double semicolons? One semicolon is used only for margin comments.

Reply:
Sure, will do